staker/staker.go

```diff
@@ -417,11 +362,85 @@ func (s *Staker) Initialize(ctx context.Context) error {
 
 		return s.blockValidator.InitAssumeValid(stakedInfo.AfterState().GlobalState)
 	}
+	return s.setupFastConfirmation(ctx)
+}
+
+// setupFastConfirmation sets the enableFastConfirmation and fastConfirmSafe variables of staker
+// based on the config, the wallet address, and the on-chain rollup designated fast confirmer.
+// Before this function, both variables should be their default (i.e. fast confirmation is disabled).
+func (s *Staker) setupFastConfirmation(ctx context.Context) error {
+	if !s.config.EnableFastConfirmation || s.wallet.Address() == nil {
+		return nil
```

Comment:
Return an error if EnableFastConfirmation but no wallet. (at least, log an error)

Reply:
EnableFastConfirmation is on by default, so this would log an error for all watchtower validators